packages/dashboard-v2/src/components/forms/LoginForm.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from "prop-types";
+import { Formik, Form } from "formik";
+import { Link } from "gatsby";
+import * as Yup from "yup";
+
+import HighlightedLink from "../HighlightedLink";
+import { TextField } from "../Form/TextField";
+import { Button } from "../Button";
+
+import accountsService from "../../services/accountsService";
+
+const loginSchema = Yup.object().shape({
+  email: Yup.string().required("Email is required").email("Please provide a valid email address"),
+  password: Yup.string().required("Password is required"),
+});
+
+const INVALID_CREDENTIALS_ERRORS = ["password mismatch", "user not found"];
+
+export const LoginForm = ({ onSuccess }) => (
+  <Formik
+    initialValues={{
+      email: "",
+      password: "",
+    }}
+    validationSchema={loginSchema}
+    onSubmit={async (values, { setErrors }) => {
+      try {
+        await accountsService.post("login", {
+          json: values,
+        });
+
+        onSuccess();
+      } catch (err) {
+        if (err.response) {
+          const data = await err.response.json();
+
+          if (INVALID_CREDENTIALS_ERRORS.includes(data.message)) {
+            setErrors({
+              email: "Invalid e-mail address or password",
+              password: "Invalid e-mail address or password",
```

Comment:
wondering why the error says "or" here, can't it be 
```
email: "Invalid email address"
password: "Invalid password"
```

Reply:
probably that's accounts not returning a specific error right? we should actually display the message from accounts I think since it can also say "registration is disabled at the moment" or other

Reply:
API actually does return separate messages for "user not found" and "password mismatch", but it's not a good practice to do so. API should only return something like "invalid credentials", without telling if the user exists in the database, so that it's not possible to extract a list of our users or check if somebody is using the service ([reference](https://tarunbatra.com/blog/security/Error-messages-in-login-process-Privacy-and-Security/#Protecting-user-base-information)). I posted [an issue](https://github.com/SkynetLabs/skynet-accounts/issues/171) in `accounts`.

As for the UI, I'm setting a generic message for that very reason -- but I guess it could be set on one of the fields only (or entirely below the form).

Reply:
I addressed this and other comments in 731b1b6d